static/elements/chromedash-approvals-dialog.js

```diff
@@ -230,6 +255,63 @@ class ChromedashApprovalsDialog extends LitElement {
     }
   }
 
+  renderConfigWidgets(approvalDef) {
+    const fieldId = approvalDef.id;
+    const isOpen = this.showConfigs.has(fieldId);
+    if (!isOpen) {
+      return nothing;
+    }
+
+    const config = this.configs.find(c => c.field_id == fieldId);
+    const owners = (config && config.owners || []).join(', ');
+    const nextAction = (config && config.next_action || '');
+    const additionalReview = (config && config.additional_review);
+    const offerAdditionalReview = fieldId == 2 || fieldId == 3;
+
+    const ownerWidget = html`
+      <input id="owners-${fieldId}" data-field="${fieldId}"
+             @change=${this.handleConfigChanged}
+             size=30 type="email" multiple placeholder="emails"
+             list="possible-owners-${fieldId}"
+             value="${owners}">
+      <datalist id="possible-owners-${fieldId}">
+         ${(this.possibleOwners[fieldId] || []).map(po => html`
+            <option value="${po}">
+         `)}
+      </datalist>
+    `;
+    const nextActionWidget = html`
+      <input id="next-action-${fieldId}" data-field="${fieldId}"
+             @change=${this.handleConfigChanged}
+             type=date name="next_action_${fieldId}"
+             value="${nextAction}">
+    `;
+    const additionalReviewWidget = html`
+      <input id="additional-review-${fieldId}" data-field="${fieldId}"
+             @change=${this.handleConfigChanged}
+             type=checkbox ?checked=${additionalReview}>
```

Comment:
```suggestion
             type=checkbox ? checked=${additionalReview}>
```

Reply:
The leading `?` controls whether the attribute is present or not.  It is normally written without a space.
https://lit.dev/docs/v1/components/templates/#bind-properties-to-templated-elements

Reply:
Aha I see.